VaccinationSystem/VaccinationSystem/Data/ApplicationDbContext.cs

```diff
@@ -9,5 +9,8 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
             : base(options)
         {
         }
+        public DbSet<DiseaseDto> Diseases { get; set; }
+        public DbSet<Vaccine> Vaccines { get; set; }
+        public DbSet<VaccinationStatusDto> VaccinationStatuses { get; set; }
```

Comment:
VaccStatus to będzie niezmienny enum więc nie trzeba tabelki. Baza danych takiego enuma będzie trzymała jako int w kolumnie w tabelce dla Vaccinations